packages/projects-docs/pages/learn/plans/workspace.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Teams
+authors: ['CompuIves']
+description:
+  A team makes it easier to collaborate on code with others. In a team you share
+  ownership of repositories, sandboxes and templates.
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+# Workspaces
+![Workspaces](../images/workspace-selector.png)
+
+CodeSandbox uses **Workspaces** to provide personal and collaborative experiences. You can think of workspaces as the boundaries for collaboration on CodeSandbox: use a [personal workspace](#personal-workspace) for public, individual development work, and use a [Pro workspace](#pro-workspace) for any type of work requiring privacy or collaboration with others.
+
+Every sandbox belongs to a workspace, and every branch of a repository has a separate development environment in each workspace. If you are working on something in your personal workspace and decide to collaborate with others, it may be necessary to transfer (in the case of a sandbox) or re-import (in the case of a repository) your work to a Pro workspace.
+
+## Personal Workspace
+
+By default, when you set up an account, you have a free personal workspace. This is a space to create, prototype and build anything you like. You can think of this workspace as your local development transferred to the cloud. The personal workspace contains any public sandboxes and repositories you added while active in your workspace.
```

Comment:
Not sure if the comparison with the local development is right here. Because you still get one VM per sandbox / repo branch, so it's more then your local development environment in a way